catalyst/contrib/data/nlp/dataset/__init__.py

```diff
@@ -1,5 +1,4 @@
 # flake8: noqa
-
```

Comment:
There is an empty line between `noqa` and `import`s in most of the other `__init__.py` files.
Could you please return back empty line?